zebra-consensus/src/chain.rs

```diff
@@ -130,10 +129,10 @@ where
             (Some(BlockHeight(block_height)), Some(BlockHeight(last_block_height)))
                 if (block_height > last_block_height + MAX_EXPECTED_BLOCK_GAP) =>
```

Comment:
should we implement the appropriate comparison and add traits so we can do this without unwrapping the block heights to work with inner values?

Reply:
I think that's a good idea, but it hasn't been a high priority:
https://github.com/ZcashFoundation/zebra/issues/751#issuecomment-663723115

I edited the description of #751 to reflect our preferred implementation.